lib/Controller/PageController.php

```diff
@@ -126,6 +126,7 @@ public function index(): TemplateResponse {
 				'app-version' => $this->config->getAppValue('mail', 'installed_version'),
 				'accounts' => base64_encode(json_encode($accountsJson)),
 				'external-avatars' => $this->preferences->getPreference('external-avatars', 'true'),
+				'external-optOut' => $this->preferences->getPreference('external-optOut', 'true'),
```

Comment:
how about just `collect-data` as key?

Reply:
sure, do you want to check it all and let me know if theres anything else i need to change?